seacrowd/sea_datasets/melayu_brunei/melayu_brunei.py

```diff
@@ -0,0 +1,203 @@
+    form in patientive voice clauses. One of the texts tells a folk story about Batang Dayang and other text
+    includes two episodes: Javanese runaways arriving in Weston and the origin of the name Bukau, a town near Weston.
+"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    SEACROWD_SCHEMA_NAME = TASK_TO_SCHEMA[_SUPPORTED_TASKS[0]].lower()
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=f"{_DATASETNAME}",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_{SEACROWD_SCHEMA_NAME}",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_{SEACROWD_SCHEMA_NAME}",
+            subset_id=f"{_DATASETNAME}",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "text": datasets.Value("string"),
+                }
+            )
+
+        elif self.config.schema == f"seacrowd_{self.SEACROWD_SCHEMA_NAME}":
+            features = schemas.self_supervised_pretraining.features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+        urls = [_URLS[key] for key in _URLS.keys()]
+        data_path = dl_manager.download_and_extract(urls)
+
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": data_path[0],
+                    "split": "train",
+                    "other_path": data_path[1:]
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepath: Path, split: str, other_path: List) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+
+        filepaths = [filepath] + other_path
+        data = []
+        for filepath in filepaths:
+            with open(filepath, "r") as f:
+                data.append([line.rstrip() for line in f.readlines()][0])
+
+        if self.config.schema == "source":
+            yield 0, {"id": Path(filepath).stem, "text": " ".join(data)}
+
+        elif self.config.schema == "seacrowd_ssp":
+            yield 0, {"id": Path(filepath).stem, "text": " ".join(data)}
```

Comment:
If we're returning the same schema for both `self.config.schema` types, then I don't see any need to separate it like this.

Reply:
I will merge two schemas in the next commit.